components/toolbar/index.ess

```diff
@@ -53,6 +53,8 @@
       height 36px
       order 2
       width 50%
+      flex-direction row-reverse
```

Comment:
yeah, I thought so too. There is some bug where if you use order it sorts the items in row-reverse. I'll dig a little deeper.